src/main/java/Main.java

```diff
@@ -1,5 +1,10 @@
 public class Main {
-    public static void main(String[] args) {
 
+    public static void main(String[] args) {
+        if (args.length > 0 && args[0].equals("-fileName")) {
+            DBConnector.createDatabase(args[1]);
```

Comment:
The **DBConnector** has too many obligations (create DB, table, connection, and queries to `card` table).
Can you do a decomposition?

Reply:
fixed